docs/plugins/ingest-geoip.asciidoc

```diff
@@ -203,3 +203,18 @@ Which returns:
 }
 --------------------------------------------------
 // TESTRESPONSE
+
+[[ingest-geoip-settings]]
+===== Settings
+
+The geoip processor supports the following settings:
+
+`ingest.geoip.cache_enabled`::
+
+    Whether to enable caching of results. Defaults to `false`.
```

Comment:
I think this cache is only going to be beneficial if ip addresses re-appear often enough.
I'm not sure if that is the case in most scenarios? That is why I told @danielmitterdorfer that we should disable this cache by default. Otherwise this cache will hold entries that don't get used and eventually are kicked out.

If we think that this assumption if false then I'm ok with enabling this by default.